schema.graphql

```diff
@@ -1574,24 +1574,24 @@ enum RecurringEventMutationType {
 }
 
 type SocialMediaUrls {
+  X: String
```

Comment:
The field name X is intentionally chosen because it refers to the social platform formerly known as Twitter.
The SocialMediaUrls structure stores URLs to various social media accounts, so having X as a field name correctly reflects its purpose. Additionally, since X is within the SocialMediaUrls context, it clearly indicates that it pertains to a social media platform. Given this, I believe the current naming is appropriate.